common/utils.go

```diff
@@ -1065,6 +1065,31 @@ func MarshalObjToYaml(obj runtime.Object) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// StringifyMap stringifies the map values
+func StringifyMap(inputMap map[string]interface{}) map[string]interface{} {
+	for key, value := range inputMap {
+		if value == nil {
+			inputMap[key] = nil
```

Comment:
do we need this assignment if it is already nil?

Reply:
done